src/irmin/append_only_intf.ml

```diff
@@ -16,17 +16,24 @@
 
 open Import
 open Store_properties
+module Irmin_key = Key
 
 module type S = sig
   (** {1 Append-only stores}
 
       Append-only stores are store where it is possible to read and add new
       values. *)
 
-  include Read_only.S
```

Comment:
any reason to have inlined this?

Reply:
If there was, I've forgotten it... reverted.